app/templates/account/oauth-aiyouth-view.pug

```diff
@@ -0,0 +1,27 @@
+extends /templates/base.jade
+
+block content
+
+
+  if me.isAnonymous()
+    h4.text-center 尚未登录，请点击上方按钮注册或登录
+
+  else if view.succeed
+    h3.text-center 恭喜绑定成功，请关闭当前页面
+
+  else if me.isStudent()
+    if !me.get('bindedAIYouth')
```

Comment:
This won't work, that's not a valid user schema property. I think this is missing some logic?